pkgs/applications/misc/feedbackd/default.nix

```diff
@@ -1,46 +1,63 @@
 { lib
 , stdenv
 , fetchFromGitLab
+, docbook-xsl-nons
+, gobject-introspection
+, gtk-doc
+, libxslt
 , meson
 , ninja
 , pkg-config
+, vala
 , wrapGAppsHook
 , glib
 , gsound
-, libgudev
 , json-glib
-, vala
-, gobject-introspection
+, libgudev
+, dbus
 }:
 
 stdenv.mkDerivation rec {
-  pname = "feedbackd-unstable";
-  version = "2021-01-25";
+  pname = "feedbackd";
+  version = "0.0.0+git20210426";
```

Comment:
I know other distros do this, but it breaks nixpkgs rules.
from the manual's [package naming secition](https://nixos.org/manual/nixpkgs/stable/#sec-package-naming):
```
 If a package is not a release but a commit from a repository, then the version part of the name must be the date of that (fetched) commit. The date must be in "YYYY-MM-DD" format. Also append "unstable" to the name - e.g., "pkgname-unstable-2014-09-23". 
```